### DISCUSSION TITLE
This looks cool, I have a couple of questions

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [MrScautHD/Sparkle#26] [Q&A]
This looks cool, I have a couple of questions

Does this have a editor? maybe Imgui or avalonia?
Does this support procedural system like minecraft?

**Comment by MrScautHD** [comment]
> This looks cool, I have a couple of questions
> 
> Does this have a editor? maybe Imgui or avalonia? Does this support procedural system like minecraft?

Heyy, nice to hear you like it!

It do not have a edtior like ImGUI but you can impliment that by yourself!

And what do you mean by a procedural system?

**Reply by Shadowblitz16** [reply]
@MrScautHD By procedural system I mean some api for generating procedural voxel worlds,
Specifically something that is pretty well optimized.

**Reply by Shadowblitz16** [reply]
> > This looks cool, I have a couple of questions
> > Does this have a editor? maybe Imgui or avalonia? Does this support procedural system like minecraft?
> 
> Heyy, nice to hear you like it!
> 
> It do not have a edtior like ImGUI but you can impliment that by yourself!
> 
> And what do you mean by a procedural system?

Also probably something where chunks can wrap around.

**Reply by copperdevs** [reply]
> @MrScautHD By procedural system I mean some api for generating procedural voxel worlds, Specifically something that is pretty well optimized.

This seems a little too specific for this project in my opinion

**Reply by Shadowblitz16** [reply]
> > @MrScautHD By procedural system I mean some api for generating procedural voxel worlds, Specifically something that is pretty well optimized.
> 
> This seems a little too specific for this project in my opinion

Oh I thought that's what type of project that this was. I just saw the block bench model of the car and assumed sorry.

**Reply by MrScautHD** [reply]
We try todo a marching cubes system.
The first versions of it are in the last Release
